app/2.0/docs/devguide/dom-template.md

```diff
@@ -60,106 +60,184 @@ except for testing. For caveats about defining elements in the main document,
 see [main document definitions](registering-elements#main-document-definitions).
 {.alert .alert-info}
 
-### Specify a string template
+### Return an HTMLTemplateElement {#templateobject}
 
-As an alternative to specifying the element's template in markup, you can specify a string template
-by creating a static `template` getter that returns a string.
+As an alternative to specifying the element's template in markup, you can create a static
+`template` getter that returns an HTMLTemplateElement.
```

Comment:
HTMLTemplateElement should be in code format.

The magic `Polymer.html` should be introduced, like,  "Polymer provides an `html` function that can be applied to a template literal to produce an `HTMLTemplateElement`." Or something @justinfagnani ? 

Unfortunately, ECMAScript kind of failed when it comes to terminology here. `foo` is a template literal. html`foo` is a _tagged template literal_. What exactly the `html` function is in this syntax is not specified, AFAICT, in the ECMAScript spec. It's... the function that you tag the literal with. Or something.

Sometimes I hate engineers. Just sayin'.

Anyway, point is, I'm basically just specifying a string, but presto change-o, I return a full-blown template element.